xamarin/Xamarin.Forms
# GH-8881 - Remove the line at the flyout's end

### Description of Change ###

Remove the line at the end of Flyout in Shell.

### Issues Resolved ### 

- fixes #8881


### Platforms Affected ### 

- iOS


### Before/After Screenshots ### 
Before:
![image](https://user-images.githubusercontent.com/20712372/73806101-4cb0ff00-47a7-11ea-93b6-a335388ee63f.png)

After:
![image](https://user-images.githubusercontent.com/20712372/73806090-47ec4b00-47a7-11ea-857b-1abf044feb7a.png)

### Testing Procedure ###
Open the Issue8881, in the Control Gallery, and open the Flyout. If you don't see the line at the end of Flyout we are good.

### PR Checklist ###
